### PR TITLE
Fixed SMSG_AREA_POI_UPDATE and CMSG_REQUEST_AREA_POI_UPDATE

### PR DESCRIPTION
**Pull request by funjoker**

### CODE REVIEW

**Comment by Shauren** [comment]
not very useful...

**Comment by funjoker** [comment]
Yeah, but no idea what this opcode is actually used for

File: WowPacketParserModule.V7_0_3_22248/Parsers/MiscellaneousHandler.cs [hunk 1]
```diff
@@ -333,5 +333,25 @@ public static void HandleOpenAlliedRaceDetailsGiver(Packet packet)
...
+
+        public static void ReadUnkAreaPoiUpdate(Packet packet, params object[] idx)
+        {
+            packet.ReadTime("UnkTime", idx);
+            packet.ReadInt32("UnkInt32_1", idx);    // AreaID?
+            packet.ReadInt32("UnkInt32_2", idx);
+            packet.ReadUInt32("UnkUInt32_3", idx);
+            packet.ReadUInt32("UnkUInt32_4", idx);
+        }
+
```
Comment (soulfrost) [inline WowPacketParserModule.V7_0_3_22248/Parsers/MiscellaneousHandler.cs]: packet.ReadTime("LastUpdate", idx);
packet.ReadInt32("QuestID", idx);    // AreaID?
packet.ReadInt32("Timer", idx);
packet.ReadUInt32("WorldStateVariableID", idx);
packet.ReadUInt32("WorldStateValue", idx);

Comment (soulfrost) [reply WowPacketParserModule.V7_0_3_22248/Parsers/MiscellaneousHandler.cs]: @funjoker

Comment (funjoker) [reply WowPacketParserModule.V7_0_3_22248/Parsers/MiscellaneousHandler.cs]: it's not QuestID (too short numbers, just 4 diggits)

**Pull request merged**